code/modules/holiday/holidays.dm

```diff
@@ -252,7 +252,9 @@
 	name = "Independence Day"
 	begin_day = 4
 	begin_month = JULY
-
+	lobby_music = list(
+		"https://www.youtube.com/watch?v=5uPoDNEn3I0", // americ
```

Comment:
This will be really grating once you get to the 3rd round, please add more songs.